### DISCUSSION TITLE
How to handle slot cards with interface names that depend on the slot # on a switch?

### DISCUSSION THREAD

**Discussion by metalgrid** [netbox-community/netbox#7773] [Q&A] [answered]
Hello everyone!

We have a switch with 4 slots. Each slot contains an interface card, and the switch shows the interface names depending on the card's slot in a XG[slot]/[port] fashion.
So for example, we have 2 cards installed, on slots 1 and 3 on the switch and the interface names are as follows:
XG1/1 XG1/2 XG1/3 ...
XG3/1 XG3/2 XG3/3 and so on

Currently we have added templates for cards on each slot respectively but it feels bad because we have a template for each card model for each slot it could go in. How do you approach this situation?

**Comment by candlerb** [comment, marked as answer]
If you're trying to use Device Bays (parent/child devices) for this, then You're Doing It Wrong™.  Please see discussion #7355.

**Reply by metalgrid** [reply]
Thanks for this. Somehow I managed to put in all search entries remarkably poor and missed this thread.
